projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -970,7 +970,7 @@ function get_paywall_blocks( $newsletter_access_level ) {
 		: __( 'Already a subscriber?', 'jetpack' );
 
 	$sign_in = '';
-	if ( ! is_user_logged_in() && $is_paid_post && ( new Host() )->is_wpcom_simple() ) {
+	if ( ! is_user_logged_in() && ( new Host() )->is_wpcom_simple() ) {
```

Comment:
Once we remove the wpcom-simple check, we can link Jetpack users to this branded page: https://wordpress.com/log-in/jetpack/link